specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/ddosProtectionPlan.json

```diff
@@ -0,0 +1,337 @@
+      "get": {
+        "tags": [
+          "DdosProtectionPlans"
+        ],
+        "operationId": "DdosProtectionPlans_ListAll",
+        "description": "Gets all DDoS protection plans in a subscription.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of DDoS protection plan resources.",
+            "schema": {
+              "$ref": "#/definitions/DdosProtectionPlanListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List all DDoS protection plans": {
+            "$ref": "./examples/DdosProtectionPlanListAll.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/ddosProtectionPlans": {
+      "get": {
+        "tags": [
+          "DdosProtectionPlans"
+        ],
+        "operationId": "DdosProtectionPlans_List",
+        "description": "Gets all the DDoS protection plans in a resource group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of DDoS protection plan resources.",
+            "schema": {
+              "$ref": "#/definitions/DdosProtectionPlanListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List DDoS protection plans in resource group": {
+            "$ref": "./examples/DdosProtectionPlanList.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DdosProtectionPlan": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DdosProtectionPlanPropertiesFormat",
+          "description": "Properties of the DDoS protection plan."
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/Resource"
+        }
+      ],
+      "description": "A DDoS protection plan in a resource group."
```

Comment:
Seems only properties we can set on `DdosProtectionPlan` are `location` and `name` and `tags`. Just trying confirming  my understanding.

Reply:
That's correct